test/unit/rules/no-invalid-link-title-test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-invalid-link-title',
+
+  config: true,
+
+  good: [
+    '<a href="https://myurl.com">Click here to read more about this amazing adventure</a>',
+    '{{#link-to}} click here to read more about our company{{/link-to}}',
+    '<LinkTo>Read more about ways semantic HTML can make your code more accessible.</LinkTo>',
+    '<LinkTo>{{foo}} more</LinkTo>',
+    '<a href="https://myurl.com" title="New to Ember? Read the full tutorial for the best experience">Read the Tutorial</a>',
+  ],
```

Comment:
Can you add another test case that ensures that `<div title="foo">foo</div>` is allowed?